core/src/main/resources/schema/app/20200512100924_CB-7013_Delete_old_7.1.1_templates.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-7013 Delete old 7.1.1 templates
+-- Migration SQL that makes the change goes here.
+
+UPDATE blueprint SET status = 'DEFAULT_DELETED' WHERE status = 'DEFAULT' AND name LIKE '7.1.1%';
```

Comment:
I think we don't need this anymore, the code shall hide things automatically.

Reply:
I removed the line and also fixed an issue with the loader service what I just realized. Different commit and different issue but if you want I can open two separated pull request for this.

Reply:
Thanks, but I think we don't need SQL at all

Reply:
removed